components/collapse/nz-collapse-panel.component.html

```diff
@@ -1,5 +1,12 @@
 <div role="tab" [attr.aria-expanded]="nzActive" class="ant-collapse-header" (click)="clickHeader()">
-  <i nz-icon type="right" class="arrow anticon-right" *ngIf="nzShowArrow"></i>
+  <div #iconContainer>
```

Comment:
There is not a layer here for antd. This div should be removed.

![image](https://user-images.githubusercontent.com/12122021/51222911-a23b2380-197a-11e9-8d55-25ab1e07cd57.png)

Reply:
I didn't remove it, because I needed it to wrap the template.
Do you have a better idea？